environment.yml

```diff
@@ -15,7 +15,7 @@ dependencies:
   # TODO This section is probably not very accurate right now (some dependencies should probably be in the backends files)
   - sqlalchemy>=1.3
   - dask>=2.22.0
-  - graphviz>=2.38
+  - graphviz>=2.42
```

Comment:
i would actually just fix it at 2.38 (and then open an issue and see what's wrong at leisure). this has been stable forever, must have just changed.

Reply:
The failing jobs have graphviz at 2.38.0 but the succeeding ones have it at 2.42 so I was trying >=2.42, but from the most recent CI failures here, it looks like 2.42 isn't even available for Windows. I'll give 2.38 a try, also looking at the diff between CI output now and last succeeding to see what other packages may have changed

Reply:
actually now I believe the problem is a package related to `graphviz` but not `graphviz` itself, as its version hasn't changed between the last succeeding CI outputs and now

`python-graphviz` changed from `0.15` to `0.16` so this may the issue. Will try pinning it

Reply:
ahh yep